Networking/Networking/Remote/RefundsRemote.swift

```diff
@@ -35,6 +35,29 @@ public final class RefundsRemote: Remote {
         enqueue(request, mapper: mapper, completion: completion)
     }
 
+    /// Retrieves a specific list of `Refund`s by `refundID`.
+    ///
+    /// - Note: this method makes a single request for a list of refunds.
+    ///         It is NOT a wrapper for `loadRefund()`
+    ///
+    /// - Parameters:
+    ///     - siteID: We are fetching remote refunds for this site.
+    ///     - orderID: We are fetching remote refunds for this order.
+    ///     - refundIDs: The array of refund IDs that are requested.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func loadRefunds(for siteID: Int, by orderID: Int, with refundIDs: [Int], completion: @escaping ([Refund]?, Error?) -> Void) {
+        let stringOfRefundIDs = refundIDs.map { String($0) }
```

Comment:
It seems like this (converting an array of Ints to an array of String, and then concatenating all of them by commas) is something that we might need to do more of in the future.

Maybe it's worth extracting it to an en extension of Array where Element == Int ?

Reply:
I fell down a rabbit hole trying to extract a new extension method. Going to add that in a separate PR.